companion/src/simulation/telemetrysimu.cpp

```diff
@@ -1057,3 +1057,22 @@ void TelemetrySimulator::LogPlaybackController::setUiDataValues()
     }
   }
 }
+
+void TelemetrySimulator::startTelemetry()
```

Comment:
These should be positioned up with the other `TelemetrySimulator` class functions.